modules/33_CDR/portfolio/bounds.gms

```diff
@@ -23,18 +23,30 @@ if(card(te_ccs33) eq 0,
 v33_emi.fx(t,regi,te_all33)$(not te_used33(te_all33)) = 0;
 v33_FEdemand.fx(t,regi,entyFe,entyFe2,te_all33)$(not te_used33(te_all33) and fe2cdr(entyFe,entyFe2,te_all33)) = 0;
 
-*' Bounds for DAC (cm_emiscen ne 1 avoids setting the boundary for the business-as-usual scenario)
-if (te_used33("dac") and cm_emiscen ne 1,
-    vm_cap.lo(t,regi,"dac",rlf)$(teNoTransform2rlf33("dac",rlf) AND (t.val ge max(2025,cm_startyear))) = sm_eps;
+*' Fix all CDR-related variables to zero for early time steps t< 2025 (no CDR in the real world)
+*' to reduce unnecessary freedom (and likelyhood of spontaneous solver infeasibilities)
+v33_emi.fx(t,regi,te_used33)$(t.val lt 2025) = 0.0;
+v33_FEdemand.fx(t,regi,entyFe,entyFe2,te_used33)$(fe2cdr(entyFe,entyFe2,te_used33) AND (t.val lt 2025)) = 0.0;
```

Comment:
Do we need the first condition? I think we can make our lives easier and just fix them all to zero before 2025

Reply:
I think it makes sense; otherwise, we unnecessarily blow up the dimensions. Only the energy carriers that are in that mapping are used anyways.